doc/workshop/ExternalModels/projectile.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#***************************************
+#* Simple analytic test ExternalModule *
+#***************************************
+#
+# Simulates time-dependent track of a projectile through the air from start to 0,
+#     assuming no air resistance.
+#     Inputs:
+#       (x0,y0) - initial position
+#       v0 - initial total velocity
+#       ang - angle of initial motion, in degrees, with respect to flat ground
+#     Outputs:
+#       (x,y) - vector positions of projectile in time
+#       t - corresponding time steps
+#
+import numpy as np
+
+def prange(v,th,y0=0,g=9.8):
+  """
+    Calculates the analytic range.
+    @ In, v, float, velocity of the projectile
+    @ In, th, float, angle to the ground for initial projectile motion
+    @ In, y0, float, optional, initial height of projectile
+    @ In, g, float, optional, gravitational constant (m/s/s)
+    @ Out, prange, float, range
+  """
+  return v*np.cos(th)/g * (v*np.sin(th) + np.sqrt(v*v*np.sin(th)**2+2.*g*y0))
+
+def time_to_ground(v,th,y0=0,g=9.8):
```

Comment:
camelBack.... the new NQ1 standard will require that any source code we generate must follow RAVEN standards